### PR TITLE
Update Undertaker to 1.1

### PR DESCRIPTION
**Pull request by GiedriusGrabauskas**
Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://github.com/gulpjs/undertaker#api
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

### CODE REVIEW

**Comment by mhegazy** [comment]
@tkqubo can you please review this change?

File: types/undertaker/index.d.ts [hunk 1]
```diff
@@ -1,115 +1,135 @@
...
+        label: string;
+        nodes: Node[];
+        type?: string;
+        branch?: boolean;
+    }
 }
 
-export interface Undertaker {
-    task: TaskMethod;
+declare class Undertaker {
```
Comment (onigoetz) [inline types/undertaker/index.d.ts]: Undertaker is as subclass of EventEmitter.

```
import {EventEmitter} from "events";
...
declare class Undertaker extends EventEmitter {
```

```

**Comment by GiedriusGrabauskas** [comment]
@onigoetz, thank you! 👍

**Comment by onigoetz** [comment]
Wow, that was fast :)

The rest looks good to me

**Comment by GiedriusGrabauskas** [comment]
@mhegazy, waiting for merge.

**Comment by GiedriusGrabauskas** [comment]
This PR created 22 days ago. Still waiting... 😞

**Comment by tkqubo** [comment]
Oh I'm so sorry I completely forgot to check this PR.  Of course it LGTM

**Comment by tkqubo** [comment]
👍

**Pull request merged**